src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40 * days;
+
+  if (days >= 7) {
+    return baseCost - 50;
+  } else if (days >= 3) {
+    return baseCost - 20;
+  }
```

Comment:
The `else` block is unnecessary after a `return` statement. You can remove `else` and keep the `if` condition to simplify the code.